README.md

```diff
@@ -191,6 +191,7 @@ An example [oauth-proxy.cfg](contrib/oauth-proxy.cfg.example) config file is in
 ```
 Usage of oauth2_proxy:
   -approval-prompt string: OAuth approval_prompt (default "force")
+  -auth-regex value: provide authentication for request paths that match (may be given multiple times), applied prior to skip-auth-regex
```

Comment:
I feel like we need to make more clear that when this isused the other paths will be *without* any authentication, so it "fails open". IE use it only if you really know what you are doing, Maybe we need a scarier name too ?